packages/react-api/package.json

```diff
@@ -23,7 +23,8 @@
     "lint:fix": "eslint 'src/**/*.{js,jsx}' --fix",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --collectCoverage"
+    "test:coverage": "jest --collectCoverage",
+    "precommit": "lint-staged"
```

Comment:
We have to remember to apply this to the current PR with the new workers package @AdriSolid @padawannn